cli/src/cmd/forge/build/core.rs

```diff
@@ -108,6 +108,14 @@ pub struct CoreBuildArgs {
     #[clap(help_heading = "PROJECT OPTIONS", help = "Generate build info files.", long)]
     #[serde(skip)]
     pub build_info: bool,
+
+    #[clap(
+        help_heading = "PROJECT OPTIONS",
+        help = "Output path to directory that build info files will be written to.",
+        long
+    )]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub build_info_path: Option<PathBuf>,
```

Comment:
there's a way to unify this via

```rust
 build_info: Option<Option<PathBuf>>
```

which works as flag `--build-info` but also accepts a path: `--build-info <path>`

but perhaps it's better to have both arguments?

` build_info: Option<Option<PathBuf>>` would need manual serialization in the `Provider` impl

Reply:
I pushed a fix that adds the `requires = "build-info"`. I am open to unify but think that it will add some complexity with the toml + hh plugin. What would toml config look like just turning it on without specifying a path so that the default value is used? Same for configuring via hh, would need to use a union type of bool/string. Also open to leave as is

Reply:
sg